data/mods/Tolerate_This/modinfo.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "Tolerate_This",
+    "name": "Bens GF recipies",
+    "authors": [ "TechyBen" ],
+    "maintainers": [ "TechyBen" ],
+    "description": "Some simple gluten free and lactose free alternative recipie options. NOT EXHAUSTIVE.",
+    "category": "items",
+    "dependencies": [ "dda" ]
+  }
+]
+
```

Comment:
You should remove this last hanging line as it appears to be causing the Jenkins and Travis failures.